common/autoware_auto_perception_rviz_plugin/src/object_detection/object_polygon_detail.cpp

```diff
@@ -670,30 +744,104 @@ void calc_bounding_box_orientation_line_list(
   calc_line_list_from_points(point_list, point_pairs, 2, points);
 }
 
+// void calc_2d_bounding_box_bottom_line_list(
```

Comment:
[Q] Do we need this comments?

Reply:
No, this is the old implementation. It will be removed,